test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -2145,6 +2146,171 @@ public void NonDefaultColNamesMultiClassificationOnnxConversionTest()
+            TestPipeline(mlContext, pipe, vecSamples, onnxFileName, columnsToCompare);
+
+            // Test scalar input NormalizeRobustScaling.
+            pipe = mlContext.Transforms.NormalizeRobustScaling(nameof(DataPoint.Features), centerData: fixZero);
+            onnxFileName = $"NormRobScalScalar-{fixZero}.onnx";
+            TestPipeline(mlContext, pipe, scalarSamples, onnxFileName, columnsToCompare);
+
+            Done();
+        }
+
+        /// <summary>
+        /// Helper method that takes a single IEstimator{ITransformer} and an IEnumerable{TRow} and converts the IEstimator{ITransformer} to an
+        /// EstimatorChain{ITransformer} and the IEnumerable{TRow} into an IDataView and then calls the actual testing method.
+        /// </summary>
+        /// <typeparam name="TRow">The type of the IEnumerable, will be auto detected.</typeparam>
+        /// <param name="mlContext">The MLContext</param>
+        /// <param name="pipeline">A single IEstimator{ITransformer} to be tested.</param>
+        /// <param name="data">The test data as a IEnumerable{TRow}.</param>
+        /// <param name="onnxFileName">Name to save the ONNX model file.</param>
+        /// <param name="columnsToCompare">Which columns you want to compare. This assumes that the column name in ONNX is the same as ML.Net, so only 1 name per column is provided.</param>
+        /// <param name="schemaDefinition">Optional schema definition for the IEnumerable{TRow}.</param>
+        private void TestPipeline<TRow>(MLContext mlContext, IEstimator<ITransformer> pipeline, IEnumerable<TRow> data, string onnxFileName, string[] columnsToCompare, SchemaDefinition schemaDefinition = null)
+            where TRow : class
+        {
+            var dataView = mlContext.Data.LoadFromEnumerable(data, schemaDefinition);
+            TestPipeline(mlContext, pipeline, dataView, onnxFileName, columnsToCompare);
+        }
+
+        /// <summary>
+        /// Helper method that takes a single IEstimator{ITransformer} and an IDataView and converts the IEstimator{ITransformer} to an
+        /// EstimatorChain{ITransformer} and then calls the actual testing method.
+        /// </summary>
+        /// <param name="mlContext">The MLContext</param>
+        /// <param name="pipeline">A single IEstimator{ITransformer} to be tested.</param>
+        /// <param name="dataView">The test data.</param>
+        /// <param name="onnxFileName">Name to save the ONNX model file.</param>
+        /// <param name="columnsToCompare">Which columns you want to compare. This assumes that the column name in ONNX is the same as ML.Net, so only 1 name per column is provided.</param>
+        /// <param name="schemaDefinition">Optional schema definition for the IEnumerable{TRow}.</param>
+        private void TestPipeline(MLContext mlContext, IEstimator<ITransformer> pipeline, IDataView dataView, string onnxFileName, string[] columnsToCompare, SchemaDefinition schemaDefinition = null)
+        {
```

Comment:
Is the SchemaDefinition parameter needed here?

Reply:
Nope. Its now removed.